src/pages/minjae/Main/components/Article/Article.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import "./Article.scss";
+import Feed from "./Feed";
+
+class Article extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      feedList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch("/data/mainData.json", {
+      method: "GET",
+    })
+      .then((res) => res.json())
+      .then((data) => {
+        this.setState({
+          feedList: data,
+        });
+      });
+  }
+
+  feedListConcat = () => {
+    // const feedList = [...this.state.feedList, {id: }]
+
+    const newComment = [
+      {
+        id: this.state.feedList.length + 1,
+        commentUserName: "rious275",
+        commentContent: [this.state.iptCommentValue],
+        isLiked: true,
+      },
+    ];
+    this.setState({
+      feedList: this.state.feedList.concat(newComment), //해당 리스트의 id값을 갖고있는 comment에 달아줘야됌
+      iptCommentValue: "",
+    });
```

Comment:
```suggestion
    this.setState({
      feedList: [...this.state.feedList, newComment]
      iptCommentValue: "",
    });
```
concat을 사용해서 배열의 복사를 하셨는데, 복사의 주체가 state에 있는 배열이어야 합니다!!